### PR TITLE
status: 2023q1: UFS/FFS snapshots: 13.2 was not released

### PR DESCRIPTION
**Pull request by grahamperrin**
@mckusick for your consideration. 

RELEASE did not occur during the first quarter, as originally scheduled, so ~~use future tense for release~~ describe the merge as the 2023q1 activity.

### CODE REVIEW

**Comment by lsalvadore** [comment]
I will let @mckusick decide on this, but since most likely the report will be published after the 13.2 release, which moreover already has release candidates built in March, I think this is not a big issue and the report can be kept in its original form.

**Comment by lsalvadore** [comment]
I have tried to reach @mckusick by email on the 11th of April in case he does not receive GitHub notifications, but he has not replied yet. I will leave this pull request open for now, but if we cannot get a reply soon we will resolve it by leaving the text as it is, so that it does not block the publication process.

File: website/content/en/status/report-2023-01-2023-03/ufs_snapshots.adoc [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 Contact: Marshall Kirk McKusick <mckusick@freebsd.org>
 
-The ability to make UFS/FFS filesystem snapshots when running with journaled soft updates and using them for doing background dumps on a live filesystem is available in the 13.2 release.
+The ability to make UFS/FFS filesystem snapshots when running with journaled soft updates, and using them for doing background dumps on a live filesystem, was merged to `releng/13.2` during the first quarter of 2023, and lands in FreeBSD 13.2-RELEASE.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-01-2023-03/ufs_snapshots.adoc:5]: Why `lands`? Shouldn't it be `has landed`?

**Review by lsalvadore — APPROVED** [review]
I think you can proceed with it even without @mckusick explicit approval. Now that you have rephrased it, clarity is improved without any risk of ambiguity.

**Comment by grahamperrin** [comment]
Committed locally (with a message that probably answers yesterday's question at <https://github.com/freebsd/freebsd-doc/pull/160#discussion_r1168311496>), however I couldn't push. 

I aim to fix and push this evening.

**Comment by lsalvadore** [comment]
Thanks @grahamperrin. Can this pull request be closed now? Or is something left?

File: website/content/en/status/report-2023-01-2023-03/ufs_snapshots.adoc (hunk 1 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-01-2023-03/ufs_snapshots.adoc:5]: I also used past tense [in an earlier draft](https://github.com/freebsd/freebsd-doc/pull/160/commits/cf4d5707cc0e7627a41cac6f2037e4f48a6f5cbe#diff-ab18c7b451449778d20fa3d5a180871c4ab489d0b4006bb0a11a763a31f99823R5). In doing so, I lost sight (in my mind's eye) of the main heading, and HTML title: 

> # FreeBSD Status Report First Quarter 2023

From <https://github.com/freebsd/freebsd-doc/commit/569297cf1d887f469ea60306ce05396db25934e0>: 

> … Change the wording a little, as if reviewed at the end of – still
within – the quarter. Not so much change that it appears to be a late
submission.

**Comment by grahamperrin** [comment]
<https://github.com/freebsd/freebsd-doc/pull/160#discussion_r1170768849> done (conversation resolved), closing now. Thanks!

**Pull request closed without merging**